common/src/lib.rs

```diff
@@ -5,6 +5,7 @@ pub mod merkle_tree;
 pub mod reserved;
 pub mod serde_spb;
 pub mod types;
+pub mod utils;
```

Comment:
1. It must be `test_utils` as it's not about the functionality about the crate.
2. It must be conditionally compiled using `#[cfg(test)]`

Reply:
1. Applied
2. https://github.com/rust-lang/cargo/issues/8379